doc/classes/Crypto.xml

```diff
@@ -13,12 +32,14 @@
 			<argument index="0" name="size" type="int">
 			</argument>
 			<description>
+				Generate a [PoolByteArray] of cryptographically secure random bytes with given [code]size[/code].
```

Comment:
Description should be with third-person verb, so "Generates". Same for all other descriptions.